### ISSUE TITLE
Bug: Tooltip health overwrites location etc on party mouseover.

### ISSUE THREAD

**Issue by kev41** [obble/modui#64]
When mouseover on targetframe and field it works perfect.

When mouseover party, to check where they are etc healthbar overwrites that information.

Screenshot: http://imgur.com/a/IKNY8

**Comment by ghost** [comment]
The healthbar also overrides Questie objectives.

**Comment by Svetsnaz** [comment]
Are you using it with TinyTip, because I had the same issue and now it's fixed

**Comment by ghost** [comment]
@kev41 Are you using it with TinyTip, because I had the same issue and now it's fixed

**Comment by kev41** [comment]
@Svetsnaz nope, no other tooltip addon - again, its only when I mouseover party/raid groups

**Comment by obble** [comment]
as svetsnaz says, this might be a complication or conflict with another addon ~ it doesnt explicitly have to be a tooltip mod. something like questie probably hooks the tooltip to insert its tracking info.

i'll look into it, though addon conflicts are at a relatively low priority in the short-term.
